app/views/user_mailer/challenge_assignment_notification.html.erb

```diff
@@ -30,17 +28,17 @@
     <%= index + 1 %>. <%= style_bold(prompt.title) %>
 
     <p>
-      <% if fandoms %><%= style_bold("Fandom:") %> <%= fandoms %><br><% end %>
-      <% if chars %><%= style_bold("Characters:") %> <%= chars %><br><% end %>
-      <% if ships %><%= style_bold("Relationships:") %> <%= ships %><br><% end %>
-      <% if ratings %><%= style_bold("Rating:") %> <%= ratings %><br><% end %>
-      <% if warnings %><%= style_bold("Warnings:") %> <%= warnings %><br><% end %>
-      <% if categories %><%= style_bold("Category:") %> <%= categories %><br><% end %>
-      <% if atags %><%= style_bold("Additional Tags:") %> <%= atags %><br><% end %>
-      <% if otags %><%= style_bold("Optional Tags:") %> <%= otags %><br><% end %>
-      <% if prompt.url && !prompt.url.blank? %><%= style_bold("Prompt URL:") %> <%= style_link(prompt.url, prompt.url) %><br><% end %>
+      <% if fandoms %><%= style_bold(Fandom.human_attribute_name("name_with_colon", count: prompt.any_fandom ? 1 : tag_groups["Fandom"].count)) %> <%= fandoms %><br><% end %>      
+      <% if chars %><%= style_bold(Character.human_attribute_name("name_with_colon", count: prompt.any_character ? 1 : tag_groups["Character"].count)) %> <%= chars %><br><% end %>
+      <% if ships %><%= style_bold(Relationship.human_attribute_name("name_with_colon", count: prompt.any_relationship ? 1 : tag_groups["Relationship"].count)) %> <%= ships %><br><% end %>
+      <% if ratings %><%= style_bold(Rating.human_attribute_name("name_with_colon")) %> <%= ratings %><br><% end %>
+      <% if warnings %><%= style_bold(Warning.human_attribute_name("name_with_colon", count: prompt.any_warning ? 1 : tag_groups["Warning"].count)) %> <%= warnings %><br><% end %>
+      <% if categories %><%= style_bold(Category.human_attribute_name("name_with_colon", count: prompt.any_category ? 1 : tag_groups["Category"].count)) %> <%= categories %><br><% end %>
+      <% if atags %><%= style_bold(Freeform.human_attribute_name("name_with_colon", count: prompt.any_freeform ? 1 : tag_groups["Freeform"].count)) %> <%= atags %><br><% end %>
+      <% if otags %><%= style_bold("#{t("optional_tags")}:") %> <%= otags %><br><% end %>
+      <% if prompt.url && !prompt.url.blank? %><%= style_bold("#{t("prompt_url")}:") %> <%= style_link(prompt.url, prompt.url) %><br><% end %>
```

Comment:
There's also a colon in the `prompt_url` string, so I think we'll end up with two colons here.